R/reactives.R

```diff
@@ -1583,6 +1708,20 @@ maskReactiveContext <- function(expr) {
 #' @param ignoreNULL Whether the action should be triggered (or value
 #'   calculated, in the case of \code{eventReactive}) when the input is
 #'   \code{NULL}. See Details.
+#' @param skipFirst Whether the action should be run the first time the
```

Comment:
Sorry I actually should have said ignoreInit, not initial. First and initial both sound like they could be "the first time the handler would run, skip/ignore it". But it really means "when the observer is first created/initialized, skip/ignore the handler whether it is otherwise supposed to run or not".